src/server/pegasus_server_impl.cpp

```diff
@@ -748,25 +754,24 @@ void pegasus_server_impl::on_multi_get(multi_get_rpc rpc)
             expire_count,
             filter_count,
             time_used);
-        _pfc_recent_abnormal_count->increment();
+        METRIC_VAR_INCREMENT(abnormal_read_requests);
     }
 
     if (expire_count > 0) {
-        _pfc_recent_expire_count->add(expire_count);
+        METRIC_VAR_INCREMENT_BY(read_expired_values, expire_count);
     }
     if (filter_count > 0) {
-        _pfc_recent_filter_count->add(filter_count);
+        METRIC_VAR_INCREMENT_BY(read_filtered_values, filter_count);
     }
```

Comment:
How about move the judgement into METRIC_VAR_INCREMENT_BY or counter metric?

Reply:
Check if it is zero in `METRIC_VAR_INCREMENT_BY`.